packages/utilities/exposure-scanning/fetch/index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable global-require */
+let octokit;
+
+(async () => {
+  if (process.env.GITHUB_ACTION && process.env.GITHUB_TOKEN) {
+    const { Octokit } = await import('@octokit/action');
+    octokit = new Octokit();
+  } else {
+    const { Octokit } = await import('@octokit/rest');
+    octokit = new Octokit();
+  }
+})();
+
+const fetchPrBody = async reqBody => {
+  const {
+    data: { body },
+  } = await octokit.request('GET /repos/{owner}/{repo}/pulls/{prId}', reqBody);
+  return body;
+};
+
+const fetchPrComments = async reqBody => {
+  const prComments = await octokit.request(
+    'GET /repos/{owner}/{repo}/issues/{prId}/comments',
+    reqBody,
+  );
+  return prComments.data.map(comment => ({
+    id: comment.id,
+    body: comment.body,
+  }));
+};
+
+const fetchPrReviewComments = async reqBody => {
+  const prReviewComments = await octokit.request(
+    'GET /repos/{owner}/{repo}/pulls/{prId}/comments',
+    reqBody,
+  );
+  return prReviewComments.data.map(comment => ({
+    id: comment.id,
+    body: comment.body,
+  }));
```

Comment:
Totally optional refactor - a little bit of destructuring for a cleaner look

```suggestion
  const { data } = await octokit.request(
    'GET /repos/{owner}/{repo}/pulls/{prId}/comments',
    reqBody,
  );
  return data.map(({ id, body }) => ({
    id,
    body,
  }));
```

Reply:
Yeah looks much cleaner, will do this now 👍